packages/notices/src/store/index.js

```diff
@@ -14,8 +14,6 @@ import * as selectors from './selectors';
  * Store definition for the notices namespace.
  *
  * @see https://github.com/WordPress/gutenberg/blob/HEAD/packages/data/README.md#createReduxStore
- *
- * @type {Object}
```

Comment:
This isn't really useful by itself, and when we remove it, typescript will automatically infer the entire type of the store, including actions/selectors/etc